app/workers/notify_subject_selector_of_seen_subject_worker.rb

```diff
@@ -1,6 +1,4 @@
-require 'subjects/cellect_client'
-
-class SeenCellectWorker
+class NotifySubjectSelectorOfSeenSubjectWorker
```

Comment:
the second subject seems redundant. not super keen on these names but maybe `NotifySubjectSelectorOfSeenWorker`